apps/docs/content/components/Testimonial/index.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Testimonial
+description: Use testimonial to display a quote from a customer or user.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image displaying a testimonial anatomy.](https://github.com/primer/brand/assets/912236/c9348fc5-36f4-40f8-a1ec-60bd84d37535)
+
+- Quote: The quote from the customer.
+- Name: The name of the customer.
+- Position: The position or role of the customer.
+- Avatar or Logo: Optional avatar of the customer or the company logo.
+
+## Usage
+
+Use the testimonial component to showcase a quote or a review from a customer. Use the testimonials to build credibility, trust, and provide social proof by displaying customers' sentiments about the product.
+
+Testimonials should be short and concise. Avoid using long quotes that can be hard to understand or that don't provide enough context.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/dd1a3e57-d94f-4f62-9840-67fce2cd78d2" />
+    <Caption>Use short and concise quotes.</Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/e6c7e682-9948-4238-bc74-f26d705cce65" />
+    <Caption>
+      Don't use long quotes that can be hard to read and understand.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+Avatars and logos are optional and can be used to provide more credibility about the customer or the company. When using an avatar, use an image of the customer to make the testimonial more personal. If an image is not available, use a logo or a similar visual asset that represents the customer company or the organization.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/8968b579-1ff6-490e-a5a2-2abd2da588e5" />
+    <Caption>
+      Use a real life image of the customer or the logo of the company.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/bcda5e31-1ccc-44a5-9d65-2f6aad326f4b" />
+    <Caption>
+      Don't use illustrations or generic images. Use real life images to provide trust.
```

Comment:
😂 at @simurai in this example

Reply:
lol.. Happy to lend my avatar for this case. 😂